.golangci.yml

```diff
@@ -0,0 +1,30 @@
+run:
+  deadline: 5m
+
+issues:
+  # Maximum issues count per one linter. Set to 0 to disable. Default is 50.
+  max-issues-per-linter: 0
+  # Maximum count of issues with the same text. Set to 0 to disable. Default is 3.
+  max-same-issues: 0
+
```

Comment:
Not sure how golangci will work when this is merged in master, but maybe we also need `new: true` here, so we're not buried by all of the old issues

Reply:
as far as I understand `new` means only for what is currently not committed or if there is nothing the last commit. Which is not what we want - we want  when this is a PR for it to look at what it will be merge and check against it which .. maybe golangci.com does ? maybe it just runs with `--new` by default ... Maybe they can write it somewhere on their page ... Given that this is how everybody uses it and at least in some of them they have issues in master but master is still green I would  guess golangci does the right thing™ 
Looking at https://golangci.com/r/github.com/loadimpact/k6/pulls/943 they make a patch and then analyse that ... they also git clone with depth 1 so ... maybe they are doing what `--new` will do by default ... I will need to find a way to test this ... probably with a bogus PR

Reply:
Yeah, I guess we can just wait and see what happens when we merge this (or another) PR with `master`. Worst case, we have a "broken" commit, not the end of the world...